pkg/cmd/roachtest/operations/utils.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2024 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package operations
+
+import (
+	"context"
+	gosql "database/sql"
+	"fmt"
+
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/operation"
+	"github.com/cockroachdb/cockroach/pkg/util/randutil"
+)
+
+func pickRandomDB(ctx context.Context, o operation.Operation, conn *gosql.DB) string {
+	rng, _ := randutil.NewPseudoRand()
+
+	// Pick a random table.
+	dbs, err := conn.QueryContext(ctx, "SELECT database_name FROM [SHOW DATABASES]")
+	if err != nil {
+		o.Fatal(err)
+		return ""
+	}
+	var dbNames []string
+	for dbs.Next() {
+		var dbName string
+		if err := dbs.Scan(&dbName); err != nil {
+			o.Fatal(err)
+			return ""
+		}
+		if dbName == "system" || dbName == "information_schema" || dbName == "crdb_internal" || dbName == "defaultdb" || dbName == "postgres" {
```

Comment:
I might be wrong, but this feels tailored for the index scenario? Or a comment on the method explaining why these tables are excluded would be helpful.

Reply:
I wanted to exclude any immutable/system DBs. Maybe `defaultdb` deserves to not be on here, but I've made this exclusion clearer now.